cmd/createtree/pem.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/trillian/crypto/keys"
+	"github.com/google/trillian/crypto/keyspb"
+)
+
+var (
+	pemKeyPath = flag.String("pem_key_path", "", "Path to the private key PEM file")
+	pemKeyPass = flag.String("pem_key_password", "", "Password of the private key PEM file")
+)
+
+func init() {
+	keyHandlers["PEMKeyFile"] = func() (proto.Message, error) {
```

Comment:
`keyHandlers` can't be initialized in the same file, because the PKCS#11 handler needs to be behind a build tag. It needs to use the global registry / auto-registration pattern. I can move this registry out of main.go and into its own package though.